### ISSUE TITLE
Differences in U-values between CEA Database and J. Fonseca 2015 article

### ISSUE THREAD

**Issue by AlexJew** [architecture-building-systems/CityEnergyAnalyst#2760]
**Describe the bug**
I am working on a energy demand simulation project in Lisbon and I am using archetypes that were characterised for the Portuguese building stocks. I was interested in comparing with the U values found in Switzerland. I noticed that there was quite a significant difference (particularly for ground and windows) in values between the U values for the archetypes mentioned in J. Fonseca, 2015 (https://doi.org/10.1016/j.apenergy.2014.12.068 ) and the default archetypes defined in CEA for the same construction periods. Has there been any update on values since 2015? And if so, which values should I use for my comparison? 

**Screenshots**

I have regrouped the values in an excel file. You can find it here (see sheet "Switzerland"): 
[Construction_standards.xlsx](https://github.com/architecture-building-systems/CityEnergyAnalyst/files/4746702/Construction_standards.xlsx)

Here is a screenshot of the tables of interest:
![image](https://user-images.githubusercontent.com/32110882/84044749-debc5480-a99f-11ea-9449-f776399b3a2f.png)

**Comment by daren-thomas** [comment]
@JIMENOFONSECA @gabriel-happle do you have any insights into this question?

**Comment by JIMENOFONSECA** [comment]
Well I believe the values got updated in the process of improving CEA throughout the years. I am not sure if those values are representative of the Swiss building stock anymore. in Fonseca et al.,2015 we took different sources of information to put them together, including a sample of 24 buildings in Zug, Switzerland. If any, I would recommend @AlexJew to look at other references beyond CEA and fonseca et al., 2015 for such a comparison.
